NYTPhotoViewer/NYTPhotoViewerArrayDataSource.m

```diff
@@ -1,20 +1,14 @@
 //
-//  NYTPhotosDataSource.m
+//  NYTPhotoViewerArrayDataSource.m
 //  NYTPhotoViewer
 //
 //  Created by Brian Capps on 2/11/15.
 //
```

Comment:
The [license](https://github.com/NYTimes/NYTPhotoViewer/blob/develop/LICENSE.md) (which was reviewed by the legal team a while back, I think) includes a copyright so I'll add one here.